bin/configurer/utils/idam-lease-user-token.sh

```diff
@@ -2,21 +2,11 @@
 
 set -eu
 
-if [[ "${IDAM_VERSION:-tactical}" == 'tactical' ]]; then
-  id=${1}
-  role=${2}
+importerUsername=${CCD_CONFIGURER_IMPORTER_USERNAME:-ccd.docker.default@hmcts.net}
```

Comment:
The same here, IMO it should not use defaults for critical arguments to limit potential surprise especially that scripts does not really report what was created.

Reply:
I have extracted all parameters for consistency